core-domain/src/main/java/org/uniprot/core/unirule/impl/PositionalFeatureImpl.java

```diff
@@ -22,6 +22,12 @@
 
     PositionalFeatureImpl(
             Range position, String pattern, boolean inGroup, String value, String type) {
+
+        if (Objects.isNull(position)) {
+            throw new IllegalArgumentException(
+                    "position is mandatory param for PositionalFeature entry");
```

Comment:
same comment for all Exceptions being thrown.
param -> parameter (just to be clear and it then also looks a bit more professional to the client, whoever that is)